pistis.go

```diff
@@ -171,6 +177,11 @@ func logic() {
 		Info("Reading commit %s", hash)
 
 		pgpObj, verifyErr := commit.Verify(keyring)
+		Debug("Author %s", commit.Author)
+		Debug("Committer %s", commit.Committer)
+		Debug("Signature %s", commit.PGPSignature)
+		Debug("pgpObj %s %v", pgpObj, pgpObj)
```

Comment:
To me `pgpObj` seems to be lots of low level numbers filling up my screen. Is this more useful than I think?

Reply:
That's why it's behind debug, it could be `trace` level if we have that. It has been useful to check the structure of objects at that point.

Reply:
Ok, I agree it would be better suited for `trace`, the other lines are fine with `debug`. We can implement trace separately.